src/sentry/models/dashboard.py

```diff
@@ -34,12 +35,22 @@ class DashboardFavoriteUser(DefaultFieldsModel):
     __relocation_scope__ = RelocationScope.Organization
 
     user_id = HybridCloudForeignKey("sentry.User", on_delete="CASCADE")
+    organization = FlexibleForeignKey("sentry.Organization", null=True)
```

Comment:
Wondering if it makes sense to link to the `OrganizationMember` instead of having both these columns?

That way, if their membership to an org is removed, these rows will also be removed.